docs/upgrade/phpstan-level-1.md

```diff
@@ -0,0 +1,63 @@
+# Upgrade Instructions for Upgrading PHPStan to Level 1
+
+This article describes the upgrade instructions for [#1040 Upgrading PHPStan](https://github.com/shopsys/shopsys/pull/1040).
+Upgrade instructions are in a separate article because there is a lot of instructions and we don't want to jam the [UPGRADE-v7.3.0](/docs/upgrade/UPGRADE-v7.3.0.md).
+Instructions are meant to be followed when you upgrade from `v7.2.0` to `v7.3.0`.
+We recommend upgrading to level 1 to .
+
+To upgrade level of PHPStan you need to:
+- change the value of `phpstan.level` Phing property in your `build.xml` (`1` is the default value)
+    ```xml
+    <property name="phpstan.level" value="1"/>
+    ```
+    - if you have [overridden the `phpstan` Phing target](/docs/introduction/console-commands-for-application-management-phing-targets.md#customization-of-phing-targets-and-properties) or don't use the `build.xml` from `shopsys/framework` package yet, look for `<arg value="--level=0"/>` in your `build.xml` and change its value instead.
+- add `phpstan-doctrine` and `phpstan-phpunit` extension packages as dev dependencies to `composer.json`
+    ```diff
+    "require-dev": {
+        "phpstan/phpstan": "^0.11",
+        "phpunit/phpunit": "^7.0",
+        "shopsys/coding-standards": "dev-master",
+        "shopsys/http-smoke-testing": "dev-master",
+    +   "phpstan/phpstan-doctrine": "^0.11.2",
+    +   "phpstan/phpstan-phpunit": "^0.11.2"
+    ```
+- include PHPStan extensions in `phpstan.neon`
+    ```diff
+        parameters:
+            ignoreErrors:
+                # Add ignored errors here as regular expressions, e.g.:
+                # - '#PHPUnit_Framework_MockObject_MockObject(.*) given#'
+                - '#Undefined variable: \$undefined#'
+    +   includes:
+    +       - vendor/phpstan/phpstan-doctrine/extension.neon
+    +       - vendor/phpstan/phpstan-phpunit/extension.neon
+    ```
+- after upgrading level of PHPStan you can expect that PHPStan will start reporting errors when running phing target `php phing phpstan`, you can get inspiration on how to fix them in [this commit.]()
```

Comment:
"this commit" is missing link

Reply:
removed as there is no commit that could really help